regulations/static/regulations/css/less/typography.less

```diff
@@ -248,17 +248,21 @@ a:visited {
 .definition:visited {
     color: @black;
     text-decoration: none;
-    border-bottom: 1px solid #d0d0ce;
+    border-bottom: 1px solid #D1D3D5
```

Comment:
This is fixed in the latest push. I also switched these to use the variables, because there's no reason for them not to. Thanks for catching @KimberlyMunoz!